src/app.d.ts

```diff
@@ -1,4 +1,5 @@
 /// <reference types="@sveltejs/kit" />
+/// <reference types="user-agent-data-types" />
```

Comment:
Out of curiosity, have you added this line manually?

Reply:
My editor wasn't resolving the types and making the types visible is a step of the installation guide: https://github.com/lukewarlow/user-agent-data-types#make-types-visible-globally-in-all-source-files-within-a-project